ivadomed/config/config.json

```diff
@@ -40,12 +47,12 @@
         "test_fraction": 0.2
     },
     "training_parameters": {
-        "batch_size": 18,
+        "batch_size": 32,
         "loss": {
             "name": "DiceLoss"
         },
         "training_time": {
-            "num_epochs": 100,
+            "num_epochs": 15,
```

Comment:
same comment as https://github.com/ivadomed/ivadomed/pull/1069/files#r826242733

Reply:
Again, no specific reason here as well. Waiting for 15 epochs is a lot lesser time than waiting for 100 epochs to know whether everything is running fine and not crashing anywhere during/after training. Well, 15 and not 5 because, segmentation outputs are not good for the latter (because too less training).